### PR TITLE
Add object level authorization#160

### PR DESCRIPTION
**Pull request by KevinFHartmann**

### CODE REVIEW

File: score-server/src/test/java/bio/overture/score/server/metadata/MetadataServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package bio.overture.score.server.metadata;
+
+public class MetadataServiceTest {
```
Comment (andricDu) [inline score-server/src/test/java/bio/overture/score/server/metadata/MetadataServiceTest.java]: why empty?

**Comment by andricDu** [comment]
Calling all @rtisma for review

File: docker/score-client-init/TEST-CA/analysis.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (rtisma) [inline docker/score-client-init/TEST-CA/analysis.json]: delete this file, since it is not used anywhere

**Comment by rtisma** [comment]
This PR also brought to light some decisions in the past that we dont want anymore, such as case insensitive scope:
https://github.com/overture-stack/score/issues/185

**Pull request merged**